### PR TITLE
add document for system variable tidb_read_consistency

### PR DESCRIPTION
**Pull request by cfzjywxk**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -849,6 +849,16 @@ Constraint checking is always performed in place for pessimistic transactions (d
...
 > - Enabling this parameter only means that Async Commit becomes an optional mode of transaction commit. In fact, the most suitable mode of transaction commit is determined by TiDB.
 
+
+### tidb_read_consistency <span class="version-mark">New in v5.4.0</span>
+
+- Scope: SESSION
+- Persists to cluster: No
+- Type: String
+- Default value: `strict`
+- This variable is used to control the read consistency for an auto-commit read statement. If the value is set to `weak`, the transaction semantics like atomicity and distributed consistency like linearizability are not guaranteed. The locks encountered by the read statment would be skipped directly and the read execution could be faster. For user scenario the auto-commit read needs to return fast and weak consistency read results are acceptable. The weak consistency read mode could be used.
```
Comment (you06) [inline system-variables.md:859]:
```suggestion
- This variable is used to control the read consistency for an auto-commit read statement. If the value is set to `weak`, the transaction semantics like atomicity and distributed consistency like linearizability are not guaranteed. The locks encountered by the read statement would be skipped directly and the read execution could be faster. For user scenario the auto-commit read needs to return fast and weak consistency read results are acceptable. The weak consistency read mode could be used.
```

File: system-variables.md [hunk 2]
```diff
@@ -849,6 +849,16 @@ Constraint checking is always performed in place for pessimistic transactions (d
...
+
+### tidb_read_consistency <span class="version-mark">New in v5.4.0</span>
+
+- Scope: SESSION
+- Persists to cluster: No
+- Type: String
+- Default value: `strict`
+- This variable is used to control the read consistency for an auto-commit read statement. If the value is set to `weak`, the transaction semantics like atomicity and distributed consistency like linearizability are not guaranteed. The locks encountered by the read statment would be skipped directly and the read execution could be faster. For user scenario the auto-commit read needs to return fast and weak consistency read results are acceptable. The weak consistency read mode could be used. 
+
+
```
Comment (you06) [inline system-variables.md:861]: Remove the second empty line.

**Review by you06 — APPROVED** [review]
LGTM, is this variable need to be changed to visible in TiDB?

**Comment by cfzjywxk** [comment]
> LGTM, is this variable need to be changed to visible in TiDB?

Yes, it's verified.  
@songrijie  PTAL

File: system-variables.md [hunk 3]
```diff
@@ -849,6 +849,15 @@ Constraint checking is always performed in place for pessimistic transactions (d
 > - If you have enabled TiDB Binlog, enabling this variable cannot improve the performance. To improve the performance, it is recommended to use [TiCDC](/ticdc/ticdc-overview.md) instead.
 > - Enabling this parameter only means that Async Commit becomes an optional mode of transaction commit. In fact, the most suitable mode of transaction commit is determined by TiDB.
 
+
```
Comment (TomShawn) [inline system-variables.md:852]:
```suggestion
```

File: system-variables.md [hunk 4]
```diff
@@ -849,6 +849,15 @@ Constraint checking is always performed in place for pessimistic transactions (d
...
+### tidb_read_consistency <span class="version-mark">New in v5.4.0</span>
+
+- Scope: SESSION
+- Persists to cluster: No
+- Type: String
+- Default value: `strict`
+- This variable is used to control the read consistency for an auto-commit read statement. If the value is set to `weak`, the transaction semantics like atomicity and distributed consistency like linearizability are not guaranteed. The locks encountered by the read statement would be skipped directly and the read execution could be faster. For user scenario the auto-commit read needs to return fast and weak consistency read results are acceptable. The weak consistency read mode could be used.
```
Comment (TomShawn) [inline system-variables.md:859]:
```suggestion
+ This variable is used to control the read consistency for an auto-commit read statement.
+ If the variable value is set to `weak`, the locks encountered by the read statement are skipped directly and the read execution might be faster, which is the weak consistency read mode. However, the transaction semantics (such as atomicity) and distributed consistency (such as linearizability) are not guaranteed.
+ For user scenarios where the auto-commit read needs to return fast and weak consistency read results are acceptable, you can use the weak consistency read mode.
```

**Comment by TomShawn** [comment]
@cfzjywxk Could you please sign the [Contributor License Agreement](https://cla-assistant.io/pingcap/docs?pullRequest=9897) as required by https://github.com/pingcap/docs/pull/9897#issuecomment-1210078760? Thanks~

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by songrijie** [comment]
LGTM

File: system-variables.md [hunk 5]
```diff
@@ -849,6 +849,16 @@ Constraint checking is always performed in place for pessimistic transactions (d
...
 
+### tidb_read_consistency <span class="version-mark">New in v5.4.0</span>
+
+- Scope: SESSION
+- Persists to cluster: No
+- Type: String
+- Default value: `strict`
++ This variable is used to control the read consistency for an auto-commit read statement.
++ If the variable value is set to `weak`, the locks encountered by the read statement are skipped directly and the read execution might be faster, which is the weak consistency read mode. However, the transaction semantics (such as atomicity) and distributed consistency (such as linearizability) are not guaranteed.
++ For user scenarios where the auto-commit read needs to return fast and weak consistency read results are acceptable, you can use the weak consistency read mode.
```
Comment (TomShawn) [inline system-variables.md:860]:
```suggestion
- This variable is used to control the read consistency for an auto-commit read statement.
- If the variable value is set to `weak`, the locks encountered by the read statement are skipped directly and the read execution might be faster, which is the weak consistency read mode. However, the transaction semantics (such as atomicity) and distributed consistency (such as linearizability) are not guaranteed.
- For user scenarios where the auto-commit read needs to return fast and weak consistency read results are acceptable, you can use the weak consistency read mode.
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**